pages/writing/writing.md

```diff
@@ -176,18 +178,26 @@ The available functionalities should be roughly similar to those of VSCode, at l
 You can execute a Julia script from your terminal, but in most cases that is not what you want to do.
 
 ```bash
-julia myfile.jl
+julia myfile.jl  # avoid this
 ```
 
 Julia has a rather high startup and compilation latency.
 If you only use scripts, you will pay this cost every time you run a slightly modified version of your code.
 That is why many Julia developers fire up a REPL at the beginning of the day and run all of their code there, chunk by chunk, in an interactive way.
+Full files can be run interactively from the REPL with the `include` function.
+
+```julia-repl
+julia> include("myfile.jl")
+```
+
+Alternatively, `includet` from the [Revise](https://timholy.github.io/Revise.jl/stable/user_reference/#Revise.includet) package can be used to "include and track" a file.
+This will automatically update changes to function definitions in the file in the running REPL session.
```

Comment:
I thought it important to show `include` as an alternative here. The Revise version followed from there since many people use it, but maybe you want to avoid `includet` in favor of using Revise with packages as discussed under Development Workflow.

Reply:
keeping it